Gemfile

```diff
@@ -5,7 +5,9 @@ ruby '2.3.1'
 gem 'aws-sdk', '~> 2'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.6'
-# Use postgresql as the database for Active Record
+# Use mysql as the database for remote database
```

Comment:
If we use mysql in production, it makes sense to use the same for local DB.

Reply:
We will need both. For local for auth tokens and similar data and remote for actual data.

Reply:
Makes sense. Maybe we could throw in some tests with the remote DB or shall we do it in a separate PR? I could extend docker compose config with a local mysql DB for testing if you write some test cases that do use the second DB.

Reply:
All tests that refers to entities that are stored in remote database were updated